packages/test-routes/src/RouteTest.tsx

```diff
@@ -12,7 +11,7 @@ interface Connection {
 }
 
 export abstract class RouteTest extends BasicTest {
-  abstract route: typeof Route
+  abstract route: any
```

Comment:
This is where the actual dependency is now cut off between production code and tests